src/plugins/content_management/public/components/card_container/card_embeddable.tsx

```diff
@@ -30,18 +31,21 @@ export class CardEmbeddable extends Embeddable<CardEmbeddableInput> {
       ReactDOM.unmountComponentAtNode(this.node);
     }
     this.node = node;
-    ReactDOM.render(
-      <EuiCard
-        textAlign="left"
-        title={this.input.title ?? ''}
-        description={this.input.description}
-        display="plain"
-        onClick={this.input.onClick}
-        icon={this.input?.getIcon?.()}
-        footer={this.input?.getFooter?.()}
-      />,
-      node
-    );
+
+    const cardProps: EuiCardProps = {
+      ...(this.input.cardProps ? { ...this.input.cardProps } : {}),
```

Comment:
```suggestion
      ...this.input.cardProps,
```
should be good enough.